flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/ScheduledNotificationBootReceiver.java

```diff
@@ -9,9 +9,12 @@
     @Override
     public void onReceive(final Context context, Intent intent) {
         String action = intent.getAction();
-        System.out.println("rebooted");
-        if (action != null && action.equals(android.content.Intent.ACTION_BOOT_COMPLETED)) {
-            FlutterLocalNotificationsPlugin.rescheduleNotifications(context);
+        System.out.println("ScheduledNotificationBootReceiver is running");
```

Comment:
can you remove this line? I actually removed in another branch that is used for the current 1.5.0 beta released